src/sql/workbench/contrib/notebook/test/browser/notebookEditor.test.ts

```diff
@@ -155,19 +194,230 @@ suite('Test class NotebookEditor', () => {
 		assert.strictEqual(currentPosition, selectedRange, 'notebookEditor.getPosition() should return the range that was selected');
 	});
 
-	// NotebookEditor-getCellEditor tests.
 	['', undefined, null, 'unknown string', /*unknown guid*/generateUuid()].forEach(input => {
-		test(`NotebookEditor: Negative Test for getCellEditor() - returns undefined for invalid or unknown guid:'${input}'`, async () => {
-			const result = notebookEditor.getCellEditor(input);
-			assert.strictEqual(result, undefined, `notebookEditor.getCellEditor() should return undefined when invalid guid:'${input}' is passed in for a notebookEditor of an empty document.`);
+		test(`NotebookEditor: Negative Test -> getCellEditor() returns undefined for invalid or unknown guid:'${input}'`, async () => {
+			await setupNotebookEditor(notebookEditor, untitledNotebookInput);
+			const inputGuid = <string>input;
+			const result = notebookEditor.getCellEditor(inputGuid);
+			assert.strictEqual(result, undefined, `notebookEditor.getCellEditor() should return undefined when invalid guid:'${inputGuid}' is passed in for a notebookEditor of an empty document.`);
 		});
 	});
 
-	test('NotebookEditor: Positive Test for getCellEditor() - returns a valid text editor object for valid guid input', async () => {
+	test('NotebookEditor: Positive Test -> getCellEditor() returns a valid text editor object for valid guid input', async () => {
```

Comment:
I really don't think the whole "positive test"/"negative test" thing is necessary. It just adds clutter and doesn't really help anything.

Reply:
removed